redis-persistence/src/main/java/com/netflix/conductor/redis/dynoqueue/RedisPinger.java

```diff
@@ -42,7 +42,7 @@ public class RedisPinger {
      * @return boolean ping result
      */
     public boolean pingWithRetry(Host host) {
-        ExponentialBackOff backOff = new ExponentialBackOff();
+        ExponentialBackOff backOff = new ExponentialBackOff(3, 1.5);
```

Comment:
we need to provide initial Interval in millis. 
https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/util/backoff/ExponentialBackOff.html#%3Cinit%3E(long,double)

Reply:
done